code/hpke/Hacl.HPKE.Curve51_CP128_SHA256.fst

```diff
@@ -0,0 +1,20 @@
+module Hacl.HPKE.Curve51_CP128_SHA256
```

Comment:
HPKE really shows why having all these explicit bundle files is quite tedious. Could we automatically generate them?